contract/test/test-postalSvc.js

```diff
@@ -28,8 +27,50 @@ import {
   makeAgoricNames,
 } from '../tools/ui-kit-goals/name-service-client.js';
 
-/** @type {import('ava').TestFn<Awaited<ReturnType<makeTestContext>>>} */
-const test = anyTest;
+/**
+ * @import {CachedBundle} from './boot-tools.js';
+ * @import {RemotableBrand} from '@endo/eventual-send';
+ * @import {WellKnown} from './market-actors.js';
+ * @import {TestFn} from 'ava';
+ * @import {makeNodeBundleCache} from '@endo/bundle-source/cache.js';
+ * @import {ExecSync} from '../tools/agd-lib.js';
+ * @import {MockWallet} from './wallet-tools.js';
+ */
+
+/** @typedef {Awaited<ReturnType<makeNodeBundleCache>>} BundleCache */
+
+/**
+ * @typedef {{
```

Comment:
the work is done now so okay to leave, but we generally define the test context by whatever the return value is of a function that makes the context.